base/number.jl

```diff
@@ -1,7 +1,19 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
 ## generic operations on numbers ##
+"""
+    isinteger(x) -> Bool
+
+Test whether `x` or all its elements are numerically equal to some integer.
```

Comment:
Does seem unnecessary/redundant and a bit confusing to have that method now that `all(isinteger, x)` should be just as fast. Guess this case has more in common with `maxabs2` etc than it does with the vectorized functions.